### PR TITLE
Fix 1788

### PR DESCRIPTION
**Pull request by charleskawczynski**
Closes https://github.com/CliMA/ClimaAtmos.jl/issues/1788

### CODE REVIEW

**Comment by LenkaNovak** [comment]
In fact, @valeriabarra, this works too! :) Thanks, @charleskawczynski !

File: src/solver/types.jl [hunk 1]
```diff
@@ -325,6 +325,7 @@ end
 function AtmosConfig(
     s = argparse_settings();
     parsed_args = parse_commandline(s),
+    comms_ctx = get_comms_context(parsed_args),
```
Comment (valeriabarra) [inline src/solver/types.jl:328]: Hey @charleskawczynski ! Thanks for the prompt fix! Did you see #1790 ? We probably both opened a PR in the same exact moment! :laughing: 

@LenkaNovak and I pushed that fix in #1790 together. If you guys think that this one is better and supersedes #1790, then please go ahead and close that one. I am going to be off starting shortly. Thank you.

**Review by valeriabarra — APPROVED** [review]

**Review by LenkaNovak — APPROVED** [review]

File: src/solver/types.jl (hunk 1 above)
Comment (valeriabarra) [reply src/solver/types.jl:328]: Okay, I closed #1789  as superseded. Thank you both.

Comment (charleskawczynski) [reply src/solver/types.jl:328]: Oh, I didn't see it, sorry! Either one is fine!

**Pull request merged**